packages/example-app-base/src/utilities/parser/Parse.ts

```diff
@@ -27,12 +27,13 @@ export function parse(source: string, propsInterfaceOrEnumName?: string): IPrope
   let propertyNameSuffix = (type: string) => (type === 'interface' ? ' interface' : ' enum');
   let propertyType = (type: string) => (type === 'interface' ? PropertyType.interface : PropertyType.enum);
 
+  // Remove all backslashes that are not immediately followed by another backslash
+  // E.g. "\text" becomes "text", "\\text" becomes "\text"
+  const escapeBackslashes = source.replace(/\\(?!\\)/g, '');
```

Comment:
nit: `escapedSource` might be a better variable name - escapeBackslashes sound like a method or RegExp instance name